services/shotgrid_common/ayon_shotgrid_hub/update_from_shotgrid.py

```diff
@@ -214,17 +210,21 @@ def create_ay_entity_from_sg_event(
 
 def update_ayon_entity_from_sg_event(
     sg_event,
+    sg_project,
     sg_session,
     ayon_entity_hub,
+    sg_enabled_entities,
     project_code_field,
-    custom_attribs_map
+    custom_attribs_map=None,
```

Comment:
is it possible that we wouldn't pass anything for `custom_attribs_map` arg? does it need to be optional?

Reply:
[a4514d7](https://github.com/ynput/ayon-shotgrid/pull/91/commits/a4514d7c4d895a5cc8ab1a875d8c0d280d6eee57)